src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/permute_slice_n_interpolation.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "permute_slice_n_interpolation.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/interpolate.hpp"
+#include "openvino/op/slice.hpp"
+#include "openvino/op/transpose.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+
+#include "itt.hpp"
+#include "utils/general_utils.h"
+
+
+namespace ov {
+
+intel_cpu::PermuteSliceAndInterpolation::PermuteSliceAndInterpolation() {
+    MATCHER_SCOPE(PermuteSliceAndInterpolation);
+
+    element::TypeVector param_precisions{ element::i8, element::u8 };
+    auto input_m = pass::pattern::wrap_type<op::v0::Parameter>(pass::pattern::type_matches_any(param_precisions));
+    auto const_m = pass::pattern::wrap_type<op::v0::Constant>();
+    auto slice_m = pass::pattern::wrap_type<op::v8::Slice>({input_m, const_m, const_m, const_m, const_m}, ov::pass::pattern::consumers_count(1));
+    auto transpose_m = pass::pattern::wrap_type<op::v1::Transpose>({slice_m, const_m});
+    auto interpolate_m = pass::pattern::wrap_type<op::v0::Interpolate,
+                                                  op::v4::Interpolate,
+                                                  op::v11::Interpolate>({transpose_m, const_m, const_m, const_m});
```

Comment:
We can transform only a sequence in which all the affected ops have only one consumer
```suggestion
    auto transpose_m = pass::pattern::wrap_type<op::v1::Transpose>({slice_m, const_m}, ov::pass::pattern::consumers_count(1));
    auto interpolate_m = pass::pattern::wrap_type<op::v0::Interpolate,
                                                  op::v4::Interpolate,
                                                  op::v11::Interpolate>({transpose_m, const_m, const_m, const_m}, ov::pass::pattern::consumers_count(1));
```

Reply:
Done